plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2754,7 +2762,9 @@ protected FeaturesDef createFeaturesDef(Map<String, String> customParams, boolea
         FeaturesDef features = new FeaturesDef();
         features.addFeatures(PAE);
         features.addFeatures(APIC);
-        features.addFeatures(ACPI);
+        if (!"s390x".equals(System.getProperty("os.arch"))) {
```

Comment:
may be better to create a method like
```
    private boolean isHostS390x() {
        return S390X.equals(System.getProperty("os.arch");
    }
```

Reply:
reused existing isGuestS390x() method for this.